tvm/src/codegen/hlsc/codegen_vhls.cc

```diff
@@ -150,20 +104,107 @@ void CodeGenVivadoHLS::VisitStmt_(const Store* op) {
     this->stream << ref
                  << "[" << PrintExpr(sb->index)
                  << "] = " << PrintExpr(sb->value) << ";\n";
-  } else if (const StreamExpr* se = op->value.as<StreamExpr>()) {
-    std::string vid = GetVarID(se->buffer_var.get()); 
-    vid = vid.substr(0, vid.find("_stream_send")); 
-    PrintIndent();
-    this->stream << vid << "["
-                 << op->index << "] = "
-                 << "fd_" << vid << ".read();\n";
   } else {
     CodeGenC::VisitStmt_(op);
   }
 }
 
+void CodeGenVivadoHLS::VisitStmt_(const Allocate* op) {
```

Comment:
Can we add more comments? It would be good to have some small examples for what would the generated code look like.